packages/components/vite.config.ts

```diff
@@ -31,7 +31,9 @@ export default defineConfig({
         environment: 'happy-dom',
         setupFiles: ['./src/setupTests.ts'],
         css: true,
+        include: ['./src/**/*.test.ts', './src/**/*.test.tsx'],
```

Comment:
The include default are good enough as you exclude in the coverage

Reply:
No, this is separate from the coverage reports. The coverage report defines which source files to ignore for coverage. This line defines which test files to ignore and not execute